projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.scss

```diff
@@ -0,0 +1,15 @@
+.aca-rules-actions-bar {
+  padding: 0 30px;
+
+  .aca-rules-breadcrumb {
```

Comment:
Any way you could try and fix the vertical alignment in the breadcrumb? If not, don't worry too much about it.